command/node/drain.go

```diff
@@ -29,6 +31,19 @@ func Drain(c *cli.Context, logger *log.Logger) error {
 	if c.Bool("force") && c.Bool("no-deadline") {
 		return fmt.Errorf("-force and -no-deadline are mutually exclusive")
 	}
+	newConstraint := &api.Constraint{}
+	if c.Bool("with-benefits") {
+		if c.String("constraint") == "" {
+			return fmt.Errorf("with-benefits selected, must provide new constrain name")
+		}
+		if c.String("operand") == "" {
+			return fmt.Errorf("with-benefits selected, must provide new constrain name")
```

Comment:
should these error messages be different than for constraint?

Reply:
they are actual for `with-benefits` flag, so I'd like to leave it like this